docs/faq/README.md

```diff
@@ -186,3 +186,77 @@ If you're interested in reading more about this, you can check out these resourc
 
 - [Why is my web component inheriting styles?](https://lamplightdev.com/blog/2019/03/26/why-is-my-web-component-inheriting-styles/)
 - [Web Fundamentals: Web Components](https://developers.google.com/web/fundamentals/web-components/shadowdom#reset)
+
+## Stubs on my element aren't called
+
+Given the following code:
+
+`my-component.js`:
+
+```js
+// example #1
+class MyComponent extends LitElement {
+  myFunction() {
+    // ...
+  }
+  render() {
+    return html`
+      <button @click=${this.myFunction}>click</button>
+    `;
+  }
+}
+```
+
+You may be surprised to find that the following test will fail, `my-component.test.js`:
+
+```js
+import { stub } from 'sinon';
+import { expect, fixture, html } from '@open-wc/testing';
+
+describe('my component', () => {
+  it('calls myFunction when a button is clicked', () => {
+    const el = fixture(
+      html`
+        <my-component></my-component>
+      `,
+    );
+    const myFunctionStub = stub(el, 'myFunction');
+    el.shadowRoot.querySelector('button').click();
+    expect(myFunctionStub).to.have.callCount(1);
+  });
+});
+```
+
+The following, however, will pass:
+
+```js
+// example #2
+render() {
+  return html`<button @click=${() => this.myFunction()}>click</button>`;
+}
+```
+
+The reason is that with `example #1` the stubbing(/wrapping) of `myFunction` happens AFTER you pass the function to lit-html, whereas with `example #2` you make fixture, stub `myFunction`, and _then_ when you click it, it will already be stubbed.
+
+How can we solve this? There are a few ways to go about this:
+
+- You can go with `@click=${this.myFunction}`, and then rerender the component in your test to make sure the function is correctly stubbed:
+
+```js
+it('does the thing', async () => {
+  const el = await fixture(
+    html`
+      <my-component></my-component>
+    `,
+  );
+  const myFunctionStub = sinon.stub(el, 'myFunction');
+  el.requestUpdate();
+  await el.updateComplete;
+  el.shadowRoot.querySelector('button').click();
+  expect(myFunctionStub).to.have.callCount(1);
+});
+```
+
+- You could pass an anonymous function that calls `this.myFunction()`, like so: `@click=${() => this.myFunction()}`. Note that this usually only makes sense if you pass some arguments to the function, e.g.: `@click=${(e) => this.myFunction(e)}`, or `@click=${() => this.myFunction(someOtherData)}`
```

Comment:
@Westbrook do you think theres a better way we can explain this?

Reply:
I'd prefer _not_ to list this as an option, and not just for the realities of that bind. Testing documentation just doesn't seem like it should be telling you how to write your functionality...

To that end, I'd suggest we list "Test Side Effects" first and then list the forced rerender approach as an "escape hatch".

Reply:
I dont consider this testing documentation as much as a technical explanation of a gotcha though, and in reality, there _can_ be uses for this kind of bind.

What if I move this point below the 'test side effects', and then add something like "As a last resort escape hatch, you could pass an anonymous ...."?